### ISSUE TITLE
[Account Linking] Infinite looping between origin and target cloud when same account who has different devices was linked in both clouds

### ISSUE THREAD

**Issue by Trishia** [plgd-dev/hub#124]
@ondrejtomcik,

As you mentioned before, there has been an  Account Linking issue related to Infinite looping between origin cloud and target cloud when the same account who has different devices was linked in both clouds each other.

I wish this issue would be solved as soon as possible.

**Comment by Trishia** [comment]
@ondrejtomcik @jkralik 
I just wondering about this issue will be solved in v2 branch or not yet ? 
please let me know.

**Comment by ondrejtomcik** [comment]
Hello Trishia
We are almost done with this huge change: https://github.com/plgd-dev/cloud/pull/176 which is going to the v2. As soon as this is merged, we are going to integrate your config.

**Comment by Trishia** [comment]
Thank you for your kind response.

**Comment by ondrejtomcik** [comment]
test

**Comment by ondrejtomcik** [comment]
test2

**Comment by Danielius1922** [comment]
Note: according to [OCF Cloud API For Cloud Services Specification](https://openconnectivity.org/specs/OCF_Cloud_API_For_Cloud_Services_Specification_v2.2.4.pdf), section 7.1:

> an Origin Cloud shall not proxy devices from the Target Cloud to another third-party cloud (whether that is an OCF Cloud or some other realization of cloud hosted functionality).

If I understood that correctly, we shouldn't forward forwarded devices in c2c-connector.

**Comment by Trishia** [comment]
Yes, you are right. You should know original devices and forwarded devices in resource directory and not offer forwarded devices info to other origin cloud.
